src/utils/cellSets/getCellSetNames.js

```diff
@@ -0,0 +1,7 @@
+const getCellSetKey = (name) => (name?.split('/')[1] || name);
+const getRootKey = (name) => name?.split('/')[0];
+
+export {
+  getCellSetKey,
+  getRootKey,
+};
```

Comment:
This file is called `getCellSetNames`, where is this format of cell set name defined? Is it used only in differential expression? Do you know why does this format exist?

Reply:
Hmm... yea, maybe the file name should be `getCellSetKeys`. I'll change it.

Reply:
The format of the cell set name is 'cellClassKey/cellSetKey'. I've adjusted the function name accordingly and put in comment too.